src/libstore/local-store.cc

```diff
@@ -890,48 +897,48 @@ void LocalStore::queryPathInfoUncached(const StorePath & path,
 
 std::shared_ptr<const ValidPathInfo> LocalStore::queryPathInfoInternal(State & state, const StorePath & path)
 {
-    /* Get the path info. */
```

Comment:
Nit: Most of the diff in this file from this point on seems weirdly spaced to me.